docs/reference/query-dsl/fuzzy-query.asciidoc

```diff
@@ -1,75 +1,99 @@
 [[query-dsl-fuzzy-query]]
 === Fuzzy Query
 
-The fuzzy query uses similarity based on Levenshtein edit distance.
+Returns documents that contain terms similar to the search term.
```

Comment:
as "similar" may mean different things even within ES, may be better to say "contain terms similar to the search term as measured by Levenshtein edit distance"

Reply:
Fixed with 9b97c9e3c00.